activerecord/test/cases/bind_parameter_test.rb

```diff
@@ -238,8 +238,13 @@ def assert_bind_params_to_sql
           #
           #   SELECT `authors`.* FROM `authors` WHERE `authors`.`id` IN (1, 2, 3)
           #
-          sql = "SELECT #{table}.* FROM #{table} WHERE #{pk} IN (#{bind_params(1..3)})"
+          if current_adapter?(:Mysql2Adapter)
```

Comment:
Why doesn't this apply to Trilogy too? :confused:

Reply:
Trilogy doesn't support prepared statements and at the top of this file:

https://github.com/rails/rails/blob/d86321843f48a15f926a4be6dad03c90c0404c2a/activerecord/test/cases/bind_parameter_test.rb#L9